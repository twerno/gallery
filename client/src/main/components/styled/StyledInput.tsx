import styled from 'styled-components';

export default styled.input`
    padding: 9px 18px;
    width: 100%;
    border: .75px solid #cecece;
    font-size: 1rem;

    &:focus {
        outline-color: rgb(103, 156, 226);
    }

`;
