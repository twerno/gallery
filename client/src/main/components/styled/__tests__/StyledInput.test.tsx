import * as React from 'react';
import { componentBasicTests } from "test/componentBasicTests";
import StyledInput from "../StyledInput";

describe("StyledInput", () => {

    componentBasicTests("StyledInput", <StyledInput />);
});