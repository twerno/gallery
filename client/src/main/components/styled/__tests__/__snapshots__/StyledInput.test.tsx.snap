// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledInput [StyledInput] Component basic tests matches the snapshot 1`] = `
.c0 {
  padding: 9px 18px;
  width: 100%;
  border: .75px solid #cecece;
  font-size: 1rem;
}

.c0:focus {
  outline-color: rgb(103,156,226);
}

<input
  className="c0"
/>
`;
